### ISSUE TITLE
java.lang.ArithmeticException: / by zero

### ISSUE THREAD

**Issue by madrob** [eclipse/jetty.project#6135]
While running unit tests in Apache Solr, I saw this division by zero exception (Jetty 9.4.34)

```
java.lang.ArithmeticException: / by zero
    at org.eclipse.jetty.util.Pool.acquire(Pool.java:278)
    at org.eclipse.jetty.client.AbstractConnectionPool.activate(AbstractConnectionPool.java:284)
    at org.eclipse.jetty.client.AbstractConnectionPool.acquire(AbstractConnectionPool.java:209)
    at org.eclipse.jetty.client.HttpDestination.process(HttpDestination.java:331)
    at org.eclipse.jetty.client.HttpDestination.send(HttpDestination.java:318)
    at org.eclipse.jetty.client.HttpDestination.send(HttpDestination.java:311)
    at org.eclipse.jetty.client.HttpDestination.send(HttpDestination.java:288)
    at org.eclipse.jetty.client.HttpDestination.send(HttpDestination.java:265)
    at org.eclipse.jetty.client.HttpClient.send(HttpClient.java:594)
    at org.eclipse.jetty.client.HttpRequest.send(HttpRequest.java:772)
    at org.eclipse.jetty.client.HttpRequest.send(HttpRequest.java:764)
```

**Comment by joakime** [comment]
Can you replicate this easily?
If so, please update to Jetty 9.4.39.v20210325 and try again.

**Comment by madrob** [comment]
Yep, you're right. Fixed in 447823316da54172b50d11396b1cbfb6bd432965 for 9.4.36
